.github/CONTRIBUTING.md

```diff
@@ -212,6 +213,12 @@ Check [this section](contribution/component-design.md) for a complete descriptio
 
 Please read our [guidelines for authoring commits](contribution/commits.md).
 
+## Publish releases
```

Comment:
Should we include here that the NPM_TOKEN should be defined on the env file?

Reply:
It's defined in the repo [secrets](https://github.com/kiwicom/orbit/settings/secrets/actions). I think we're using @mainframev's one for now. We can consider creating an NPM service account in the future.